SimG4Core/Application/src/RunManagerMTWorker.cc

```diff
@@ -266,17 +267,17 @@ void RunManagerMTWorker::initializeThread(RunManagerMT& runManagerMaster, const
     << "RunManagerMTWorker: start initialisation of PhysicsList for the thread";
 
   physicsList->InitializeWorker();
-  m_tls->kernel->SetPhysics(physicsList);
-  m_tls->kernel->InitializePhysics();
+  m_tls->kernel.get()->SetPhysics(physicsList);
```

Comment:
the `get()` calls should not be necessary here (or most other places in this file); `unique_ptr` passes through operator/accessor calls to the underlying pointer.

Reply:
@kpedro88 , it is possible, I use this pattern in order do not forget what type of ptr it is...